lib/core/include/scipp/core/element/util.h

```diff
@@ -95,7 +95,7 @@ constexpr auto islinspace =
                         scipp::span<const int64_t>, scipp::span<const int32_t>,
                         scipp::span<const time_point>>,
                transform_flags::expect_no_variance_arg<0>,
-               [](const units::Unit &) { return units::one; },
+               [](const units::Unit &) { return units::none; },
```

Comment:
`isarange` below appears to have the same problems. Are there more?

Reply:
:+1: I'll check.

The `isarange` seems to not be exposed to the python interface. Should it be?

Reply:
Seems it was never needed...